src/malli/malli.cljc

```diff
@@ -0,0 +1,302 @@
+(ns malli.malli)
```

Comment:
This looks really good! I think it just needs some more spikes in regard to producing (or even round tripping, which would be nice) json-schema.